edward/criticisms.py

```diff
@@ -85,32 +95,35 @@ def evaluate(metrics, model, variational, data):
         return evaluations
 
 def ppc(model, variational=None, data=Data(), T=None, size=100):
-    """
-    Posterior predictive check.
+    """Posterior predictive check.
     (Rubin, 1984; Meng, 1994; Gelman, Meng, and Stern, 1996)
-    If variational is not specified, it defaults to a prior predictive
-    check (Box, 1980).
+    If no posterior approximation is provided through ``variational``, 
+    then we default to a prior predictive check (Box, 1980).
 
     PPC's form an empirical distribution for the predictive discrepancy,
-    p(T) = \int p(T(yrep) | z) p(z | y) dz
-    by drawing replicated data sets yrep and calculating T(yrep) for
-    each data set. Then it compares it to T(y).
+
+    .. math::
+        p(T) = \int p(T(yrep) | z) p(z | y) dz
+
+    by drawing replicated data sets yrep and calculating 
+    :math:`T(yrep)` for each data set. Then it compares it to 
+    :math:`T(y)`.
 
     Parameters
     ----------
-    model : Model
-        class object with a 'sample_likelihood' method
-    variational : Variational, optional
+    model : ed.Model
+        class object that implements the ``sample_likelihood`` method
+    variational : ed.Variational, optional
         latent variable distribution q(z) to sample from. It is an
         approximation to the posterior, e.g., a variational
         approximation or an empirical distribution from MCMC samples.
-        If not specified, samples will be obtained from model
-        with a 'sample_prior' method.
-    data : Data, optional
+        If not specified, samples will be obtained from the model
+        through the ``sample_prior`` method.
+    data : ed.Data, optional
         Observed data to compare to. If not specified, will return
         only the reference distribution with an assumed replicated
         data set size of 1.
-    T : function, optional
+    T : TensorFlow function, optional
```

Comment:
I would leave above as function, which is its type. By "function written in TensorFlow", it is a function taking Tensor inputs and returns a Tensor.